models/vns_epg_def.go

```diff
@@ -0,0 +1,113 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/container"
+)
+
+const (
+	DnvnsEPgDef        = "uni/tn-%s/GraphInst_C-[%s]-G-[%s]-S-[%s]/NodeInst-%s/LegVNode-%s/EPgDef-%s"
```

Comment:
there is actually more than 1 DN so I would havoid using those DnvnsEPgDef and parentDnvns..

Reply:
ok